Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN apt-get update \
 
 VOLUME ["/usr/src/app/localData","/usr/src/app/localMetadata"]
 
+ENV NO_PROXY localhost,127.0.0.1
+ENV no_proxy localhost,127.0.0.1
```

Comment:
Just wondering, why do you need upper and lower case? :)

Reply:
It's a convention. In fact, there is a third one no_PROXY, but those 2 are the most common.